dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -151,7 +147,7 @@ const TableWithFavorite = () => {
               redirect="auth"
             />
           )}
-          <div className="table-container" variant={PageSectionVariants.light}>
+          <div className="table-container">
```

Comment:
You've now put this fix into two separate PRs, but it's breaking others as well. I suspect it would make sense to pull this out into a separate PR that we can review and merge quickly!

Reply:
Moved it to a separate PR